dbt/models/proximity/docs.md

```diff
@@ -128,6 +128,14 @@ Distance from each PIN to the nearest Metra stop.
 **Primary Key**: `pin10`, `year`
 {% enddocs %}
 
+# dist_pin_to_new_construction
+
+{% docs table_dist_pin_to_new_construction %}
+Distance from each PIN to the nearest new construction.
```

Comment:
**suggestion:** Let's add the definition of "new construction" i.e. within 3 years of the target year to the docs here so folks don't have to go digging.